llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -51,7 +51,10 @@ void ComputedShaderFlags::print(raw_ostream &OS) const {
   if (FlagVal == 0)
     return;
   OS << "; Note: shader requires additional functionality:\n";
-#define SHADER_FEATURE_FLAG(bit, FlagName, Str)                                \
+#define SHADER_FEATURE_FLAG(Bit, DxilModuleNum, FlagName, Str)                 \
+  if (FlagName)                                                                \
+    OS << ";       " Str "\n";
+#define DXIL_MODULE_FLAG(Bit, FlagName, Str)                                   \
```

Comment:
I don't think module flags should be listed as additional functionality required, but it could still be useful to list them, without implying that they are optional feature requirements.

Reply:
Added